tests/minicrater/driver.rs

```diff
@@ -123,8 +123,12 @@ impl MinicraterRun {
             .expect("failed to write copy of the json report");
 
         let changeset = Changeset::new(
-            &String::from_utf8(expected_report).expect("invalid utf-8 in the expected report"),
-            &String::from_utf8(actual_report.clone()).expect("invalid utf-8 in the actual report"),
+            &String::from_utf8(expected_report)
+                .expect("invalid utf-8 in the expected report")
+                .replace("\r\n", "\n"),
+            &String::from_utf8(actual_report.clone())
+                .expect("invalid utf-8 in the actual report")
+                .replace("\r\n", "\n"),
```

Comment:
```suggestion
```

We only want to do this for the expected report. The generated one should always have unix-style newlines.